pkgs/applications/networking/filebrowser/default.nix

```diff
@@ -0,0 +1,55 @@
+{ buildGoModule, buildNpmPackage, fetchFromGitHub, lib }:
+
+let
+  frontend = buildNpmPackage rec {
+    pname = "filebrowser-frontend";
+    version = "2.23.0";
+
+    src = fetchFromGitHub {
+        owner = "filebrowser";
+        repo = "filebrowser";
+        rev = "v${version}";
+        sha256 = "sha256-xhBIJcEtxDdMXSgQtLAV0UWzPtrvKEil0WV76K5ycBc=";
+    } + "/frontend";
+
+    npmDepsHash = "sha256-acNIMKHc4q7eiFLPBtKZBNweEsrt+//0VR6dqwXHTvA=";
+
+    NODE_OPTIONS = "--openssl-legacy-provider";
+
+    installPhase = ''
+        runHook preInstall
+
+        mkdir -p $out/dist
+        mv dist $out/
```

Comment:
Maybe keep it simple: `mv dist $out` and don't bother with creating `dist`

Reply:
Done